scripts/check_style.sh

```diff
@@ -17,14 +17,16 @@ fi
 
 FORMATERROR=$(
 (
-git grep -nIE "\<(if|for)\(" -- '*.h' '*.cpp'
-git grep -nIE "\<if\>\s*\(.*\)\s*\{\s*$" -- '*.h' '*.cpp'
+	git grep -nIE "\<(if|for)\(" -- '*.h' '*.cpp' # no space after "if" or "for"
+	git grep -nIE "\<if\>\s*\(.*\)\s*\{\s*$" -- '*.h' '*.cpp' # "{\n" on same line as "if" / "for"
+	git grep -nIE "\(const " -- '*.h' '*.cpp' # const on left side of type
```

Comment:
Thanks - I really need this check :-D.
And:
``[a-zA-Z0-9_] *&[a-zA-Z]`` or something like that maybe?

Reply:
That's something you can add :)

Reply:
Actually we could also add `,const`

Reply:
Ok, I'll add both :-).